docs/faq.rst

```diff
@@ -104,8 +126,27 @@ You must fetch the channel directly and then call the appropriate method. Exampl
     channel = client.get_channel(12324234183172)
     await channel.send('hello')
 
+How do I send a DM?
+~~~~~~~~~~~~~~~~~~~
+
+Get the :class:`User` object and call :meth:`User.send`. For example: ::
+
+	user = client.get_user(381870129706958858)
+	await user.send('👀')
+
+If the user is not cached (:meth:`Client.get_user` returns ``None``), you can use :meth:`Client.fetch_user` instead.
+
+If you are responding to an event, such as :func:`on_message`, you already have the :class:`User` object via :attr:`Message.author`..
```

Comment:
```suggestion
```

This sentence is better off being shown in the code sample rather than written text.